### ISSUE TITLE
[Bug]: Netdata Health applies template to wrong charts

### ISSUE THREAD

**Issue by Pingger** [netdata/netdata#16854]
### Bug description

I have 2 (relevant) health config files:
`/etc/netdata/health.d/ts3.conf`:
```
template: packet_loss
families: *
      on: ts3.packetloss
  lookup: average -1m unaligned foreach speech,control,keepalive
   units: %
   every: 10s
    warn: $this > 2
    crit: $this > 3
    info: packet loss percentage
      to: silent
[...]
```
`/etc/netdata/health.d/fping.conf`:
```
[...]
template: packet_loss
families: *
      on: fping.quality
  lookup: average -10m unaligned of returned
    calc: 100 - $this
   green: 2
     red: 10 
   units: %
   every: 10s
    warn: $this > $green
    crit: $this > $red
    info: packet loss percentage
   delay: up 1m multiplier 2 max 6h down 10m multiplier 2 max 12h
      to: sysadmin

```

And for some reason, the ts3 template applies to the fping charts and is overriding several values of it, despite the `on` property clearly distinguishing which charts are to be affected.
![image](https://github.com/netdata/netdata/assets/4761298/b49348ac-2f02-4507-a373-9d097c1259c5)

The issue started occuring 2 days ago (on 2024-01-24 04:00:00 UTC, right after the automatic update of the agent using the nightly build)

### Expected behavior

The alarms to stay separated and only applying to the charts they are configured for and actually respecting their settings

### Steps to reproduce

1. Use alarm configs as provided
2. Use fping module with whatever servers
3. Use ts3 with packetloss graph with whatever servers
4. notice the alarms being a weird mixture


### Installation method

kickstart.sh

### System info

```shell
Linux iskariot.info 5.4.0-167-generic #184-Ubuntu SMP Tue Oct 31 09:21:49 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
/etc/lsb-release:DISTRIB_ID=Ubuntu
/etc/lsb-release:DISTRIB_RELEASE=20.04
/etc/lsb-release:DISTRIB_CODENAME=focal
/etc/lsb-release:DISTRIB_DESCRIPTION="Ubuntu 20.04.6 LTS"
/etc/os-release:NAME="Ubuntu"
/etc/os-release:VERSION="20.04.6 LTS (Focal Fossa)"
/etc/os-release:ID=ubuntu
/etc/os-release:ID_LIKE=debian
/etc/os-release:PRETTY_NAME="Ubuntu 20.04.6 LTS"
/etc/os-release:VERSION_ID="20.04"
/etc/os-release:VERSION_CODENAME=focal
/etc/os-release:UBUNTU_CODENAME=focal
```


### Netdata build info

```shell
Packaging:                                                                                                                                                                                                                                  
    Netdata Version ____________________________________________ : v1.44.0-239-nightly                                                                                                                                                      
    Installation Type __________________________________________ : binpkg-deb                                                                                                                                                               
    Package Architecture _______________________________________ : x86_64                                                                                                                                                                   
    Package Distro _____________________________________________ :                                                                                                                                                                          
    Configure Options __________________________________________ : dummy-configure-command                                                                                                                                                  
Default Directories:                                                                                                                                                                                                                        
    User Configurations ________________________________________ : /etc/netdata                                                                                                                                                             
    Stock Configurations _______________________________________ : /usr/lib/netdata/conf.d                                                                                                                                                  
    Ephemeral Databases (metrics data, metadata) _______________ : /var/cache/netdata                                                                                                                                                       
    Permanent Databases ________________________________________ : /var/lib/netdata                                                                                                                                                         
    Plugins ____________________________________________________ : /usr/libexec/netdata/plugins.d                                                                                                                                           
    Static Web Files ___________________________________________ : /var/lib/netdata/www                                                                                                                                                     
    Log Files __________________________________________________ : /var/log/netdata                                                                                                                                                         
    Lock Files _________________________________________________ : /var/lib/netdata/lock                                                                                                                                                    
    Home _______________________________________________________ : /var/lib/netdata                                                                                                                                                         
Operating System:                                                                                                                                                                                                                           
    Kernel _____________________________________________________ : Linux                                                                                                                                                                    
    Kernel Version _____________________________________________ : 5.4.0-167-generic                                                                                                                                                        
    Operating System ___________________________________________ : Ubuntu                                                                                                                                                                   
    Operating System ID ________________________________________ : ubuntu                                                                                                                                                                   
    Operating System ID Like ___________________________________ : debian                                                                                                                                                                   
    Operating System Version ___________________________________ : 20.04.6 LTS (Focal Fossa)                                                                                                                                                
    Operating System Version ID ________________________________ : none                                                                                                                                                                     
    Detection __________________________________________________ : /etc/os-release                                                                                                                                                          
Hardware:                                                                                                                                                                                                                                   
    CPU Cores __________________________________________________ : 8                                                                                                                                                                        
    CPU Frequency ______________________________________________ : 1996000000                                                                                                                                                               
    RAM Bytes __________________________________________________ : 33659760640                                                                                                                                                              
    Disk Capacity ______________________________________________ : 858993459200
    CPU Architecture ___________________________________________ : x86_64
    Virtualization Technology __________________________________ : kvm
    Virtualization Detection ___________________________________ : systemd-detect-virt
Container:
    Container __________________________________________________ : none
    Container Detection ________________________________________ : systemd-detect-virt
    Container Orchestrator _____________________________________ : none
    Container Operating System _________________________________ : none
    Container Operating System ID ______________________________ : none
    Container Operating System ID Like _________________________ : none
    Container Operating System Version _________________________ : none
    Container Operating System Version ID ______________________ : none
    Container Operating System Detection _______________________ : none
Features:
    Built For __________________________________________________ : Linux
    Netdata Cloud ______________________________________________ : YES
    Health (trigger alerts and send notifications) _____________ : YES
    Streaming (stream metrics to parent Netdata servers) _______ : YES
    Back-filling (of higher database tiers) ____________________ : YES
    Replication (fill the gaps of parent Netdata servers) ______ : YES
    Streaming and Replication Compression ______________________ : YES (zstd lz4 gzip)
    Contexts (index all active and archived metrics) ___________ : YES
    Tiering (multiple dbs with different metrics resolution) ___ : YES (5)
    Machine Learning ___________________________________________ : YES
Database Engines:
    dbengine ___________________________________________________ : YES
    alloc ______________________________________________________ : YES
    ram ________________________________________________________ : YES
    none _______________________________________________________ : YES
Connectivity Capabilities:
    ACLK (Agent-Cloud Link: MQTT over WebSockets over TLS) _____ : YES
    static (Netdata internal web server) _______________________ : YES
    h2o (web server) ___________________________________________ : YES
    WebRTC (experimental) ______________________________________ : NO
    Native HTTPS (TLS Support) _________________________________ : YES
    TLS Host Verification ______________________________________ : YES
Libraries:
    LZ4 (extremely fast lossless compression algorithm) ________ : YES
    ZSTD (fast, lossless compression algorithm) ________________ : YES
    zlib (lossless data-compression library) ___________________ : YES
    Brotli (generic-purpose lossless compression algorithm) ____ : NO
    protobuf (platform-neutral data serialization protocol) ____ : YES (system)
    OpenSSL (cryptography) _____________________________________ : YES
    libdatachannel (stand-alone WebRTC data channels) __________ : NO
    JSON-C (lightweight JSON manipulation) _____________________ : YES
    libcap (Linux capabilities system operations) ______________ : NO
    libcrypto (cryptographic functions) ________________________ : YES
    libyaml (library for parsing and emitting YAML) ____________ : YES
Plugins:
    apps (monitor processes) ___________________________________ : YES
    cgroups (monitor containers and VMs) _______________________ : YES
    cgroup-network (associate interfaces to CGROUPS) ___________ : YES
    proc (monitor Linux systems) _______________________________ : YES
    tc (monitor Linux network QoS) _____________________________ : YES
    diskspace (monitor Linux mount points) _____________________ : YES
    freebsd (monitor FreeBSD systems) __________________________ : NO
    macos (monitor MacOS systems) ______________________________ : NO
    statsd (collect custom application metrics) ________________ : YES
    timex (check system clock synchronization) _________________ : YES
    idlejitter (check system latency and jitter) _______________ : YES
    bash (support shell data collection jobs - charts.d) _______ : YES
    debugfs (kernel debugging metrics) _________________________ : YES
    cups (monitor printers and print jobs) _____________________ : YES
    ebpf (monitor system calls) ________________________________ : YES
    freeipmi (monitor enterprise server H/W) ___________________ : YES
    nfacct (gather netfilter accounting) _______________________ : YES
    perf (collect kernel performance events) ___________________ : YES
    slabinfo (monitor kernel object caching) ___________________ : YES
    Xen ________________________________________________________ : YES
    Xen VBD Error Tracking _____________________________________ : NO
    Logs Management ____________________________________________ : YES
Exporters:
    AWS Kinesis ________________________________________________ : NO
    GCP PubSub _________________________________________________ : NO
    MongoDB ____________________________________________________ : YES
    Prometheus (OpenMetrics) Exporter __________________________ : YES
    Prometheus Remote Write ____________________________________ : YES
    Graphite ___________________________________________________ : YES
    Graphite HTTP / HTTPS ______________________________________ : YES
    JSON _______________________________________________________ : YES
    JSON HTTP / HTTPS __________________________________________ : YES
    OpenTSDB ___________________________________________________ : YES
    OpenTSDB HTTP / HTTPS ______________________________________ : YES
    All Metrics API ____________________________________________ : YES
    Shell (use metrics in shell scripts) _______________________ : YES
Debug/Developer Features:
    Trace All Netdata Allocations (with charts) ________________ : NO
    Developer Mode (more runtime checks, slower) _______________ : NO
```


### Additional info

ts3.chart.py with packetloss chart: [ts3.chart.py.txt](https://github.com/netdata/netdata/files/14068506/ts3.chart.py.txt)

**Comment by ilyam8** [comment]
@Pingger hi. I think this is a bug, but it can be worked around by keeping the template name unique - it's good practice anyway. Make it `fping_packet_loss` and `ts3_packet_loss`.

**Comment by ilyam8** [comment]
Fixed in #16862
